cli/scenarios.js

```diff
@@ -123,13 +125,20 @@ async function prepareScenarioRecords (model, scenarioFilePath) {
 
   const modelId = model.id;
 
+  const { baseYear } = model;
```

Comment:
merge with `model` above

Reply:
It's a requirement, the format adopted in scenarios files is `Pop{baseYear}` (Pop2018).

Reply:
👍